src/tests/ftest/nvme/enospace.py

```diff
@@ -94,17 +94,18 @@ def setUp(self):
         self.daos_cmd = DaosCommand(self.bin)
         self.create_pool_max_size()
 
-    def get_pool_space_metrics(self, metrics):
-        """Return the metrics on space usage.
+    def get_pool_space_metrics(self, pool, metrics):
+        """Return the metrics on space usage of a given pool.
 
         Args:
+            pool (object): target TestPool.
```

Comment:
very minor - only if you need to push again
```suggestion
            pool (TestPool): target TestPool.
```

Reply:
- [ ] Update param type

Reply:
> * [x]  Update param type

Fixed with commit 115170b98